events.go

```diff
@@ -371,7 +371,7 @@ type domainCallbackContext struct {
 
 var goCallbackLock sync.RWMutex
 var goCallbacks = make(map[int]*domainCallbackContext)
-var nextGoCallbackId int
+var nextGoCallbackId int = 5
```

Comment:
What's the 5 for? Can you make a `const` with a name so the magic number has some background for future readers?